packages/protocol/test-sol/unit/common/ProxyFactory08.t.sol

```diff
@@ -13,7 +13,7 @@ contract ProxyFactoryTest is Test, Utils08 {
 
   function setUp() public {
     proxyFactory08 = new ProxyFactory08();
-    proxyInitCode = vm.getCode(string.concat("out/Proxy.sol/Proxy.json"));
```

Comment:
this was a bug in foundry that could only use those paths and we probably have it like this elsewhere in the code

Reply:
Great point looks like it's used here: 

https://github.com/celo-org/celo-monorepo/blob/67d6acc3268f22807d9f2fc239fd2a2567a18751/packages/protocol/migrations_sol/Migration.s.sol#L131-L134

I propose this gets fixed in a separate PR to separate changes into small PRs.